packages/ogmios/test/Provider/TxSubmitProvider/OgmiosTxSubmitProvider.test.ts

```diff
@@ -101,6 +116,61 @@ describe('OgmiosTxSubmitProvider', () => {
         CardanoNodeErrors.TxSubmissionErrors.EraMismatchError
       );
     });
+
+    it('throws an error if context has handles, and no handleProvider is passed', async () => {
+      mockServer = createMockOgmiosServer({
+        submitTx: { response: { failWith: { type: 'eraMismatch' }, success: false } }
+      });
+      await listenPromise(mockServer, connection.port);
+      provider = new OgmiosTxSubmitProvider(connection, { logger });
+      await provider.initialize();
+      await provider.start();
+
+      await expect(
+        provider.submitTx({
+          context: {
+            handles: [handleResolution]
+          },
+          signedTransaction: emptyUintArrayAsHexString
+        })
+      ).rejects.toThrowError(/no handleprovider passed/i);
+    });
+
```

Comment:
```suggestion
    it('does not throw an error if handles resolve to same addresses as in context', async () => {
```

Reply:
0dfd234d683942a8f490be4b630f098e8d0fa823 should solve this